File: test/new_lang/test3/TestProgram.pas
program TestProgram;
uses SplashKit;

procedure Main();
begin
  WriteLn(GetInt());
  WriteLn(GetInt(1));
  WriteLn(GetUInt(2));
  WriteLn(GetShort(3));
  WriteLn(GetUShort(4));
  WriteLn(GetFloat(5.55):4:2);
  WriteLn(GetDouble(6.66):4:2);
  WriteLn(GetLong(7));
  WriteLn(GetChar('A'));
  WriteLn(GetUChar('a'));
  WriteLn(GetBool(true));
  WriteLn(GetBool(false));
end;

begin
  Main();
end.

File: test/new_lang/test3/TestProgram.cs
using static SplashKitSDK.SplashKit;
using static System.Console;

public class Program
{
    public static void Main()
    {
        WriteLine(GetInt());
        WriteLine(GetInt(1));
        WriteLine(GetUint(2));
        WriteLine(GetShort(3));
        WriteLine(GetUshort(4));
        WriteLine(GetFloat(5.55f));
        WriteLine(GetDouble(6.66));
        WriteLine(GetLong(-1));
        WriteLine(GetChar('A'));
        WriteLine(GetUchar('a'));
        WriteLine(GetBool(true));
        WriteLine(GetBool(false));
    }
}
